test/functional/fixtures/native-dialogs-handling/testcafe-fixtures/in-iframe.test.js

```diff
@@ -0,0 +1,53 @@
+'@fixture iframePage';
```

Comment:
other fixtures called `confirm page`, `index page` but this in camel-case. Call it `iframe page`